packages/admin-portal/src/services/apps.ts

```diff
@@ -56,7 +56,7 @@ export const fetchAppsList = async (params: FetchAppsListParams): Promise<PagedR
     return response
   } catch (error) {
     logger(error)
-    throw new Error(error)
+    throw error
```

Comment:
I think this should throw error?.response instead throw error

Reply:
@tanphamhaiduong  I disagree. Sometimes, error may not come from HTTP request, it could come from code (e.g. Fetcher broken, utilities function broken), and result `Error` instance, or the backend fail to return the correct error (fail to connect? 404 result `Error` instance. If you return `error?.response`, it won't work for cases that not our standard error object, and we end up throw `undefined`?